integration-tests/runtime-tests/src/utils/polkadotjs.ts

```diff
@@ -35,9 +35,9 @@ export function sendUnsignedAndWaitFor<T extends AnyTuple>(
   call: SubmittableExtrinsic<"promise">,
   intendedToFail:boolean
 ): Promise<IEvent<T>> {
-  return new Promise<IEvent<T>>((resolve, reject) => {
+  return new Promise<IEvent<T>>(function (resolve, reject) {
```

Comment:
The mocha documentation state that `If you do not need to use Mocha’s context, lambdas should work`. All the functions exept the top level one where we do the Mocha `this.settimeout` does not require the context? If so, why refactoring this functions as well?

Reply:
This is mainly an artifact, where I tried to call `this.skip()` within that function, to debug the errors mentioned in 3.
I just didn't revert it afterward.

Reply:
Makes sense :)!